packages/pwa/app/commerce-api/auth.js

```diff
@@ -141,18 +171,22 @@ class Auth {
             let authorizationMethod = '_loginAsGuest'
             if (credentials) {
                 authorizationMethod = '_loginWithCredentials'
-            } else if (this._authToken && this._refreshToken) {
+            } else if (this._refreshToken) {
```

Comment:
Why are we no longer verifying if the `this._authToken` exists?

Reply:
`this._authToken` should expire when session ends, but refresh token should persist for long period (90 days).